src/deepsparse/server/build_logger.py

```diff
@@ -57,7 +57,10 @@ def build_logger(server_config: ServerConfig) -> BaseLogger:
 
     loggers_config = server_config.loggers
     if not loggers_config:
-        return None
+        return AsyncLogger(
```

Comment:
Pulling default logger inside the `build_logger`. This will facilitate testing and feels more appropriate (we encapsulate all the logger creation inside one function)